src/parser/deserializer.ts

```diff
@@ -120,6 +123,49 @@ function deserializeObject(
   const promoteLongs = options['promoteLongs'] == null ? true : options['promoteLongs'];
   const promoteValues = options['promoteValues'] == null ? true : options['promoteValues'];
 
+  // Ensures default validation option if none given
+  const validation = options.validation == null ? { utf8: true } : options.validation;
+
+  // Shows if global utf8 validation is enabled or disabled
+  let globalUTFValidation = true;
+  // Reflects utf8 validation boolean regardless of global or specific key validation
+  let uniformBool: boolean;
```

Comment:
I think we need a new name for this, it is indeed the boolean calculated from looking at all the ones specified but when reading the code lower down I'm losing the meaning, this is whether or not we are by default : validating or not validating. Maybe `validationSetting`? its like more generic but kinda descriptive of the purpose?